app/controllers/nonsipp/landorpropertydisposal/LandOrPropertyDisposalListController.scala

```diff
@@ -210,7 +219,8 @@ class LandOrPropertyDisposalListController @Inject()(
                           request.userAnswers,
                           request.schemeDetails.schemeName,
                           showBackLink = true,
-                          maximumDisposalsReached = maximumDisposalsReached
+                          maximumDisposalsReached = maximumDisposalsReached,
+                          allPropertiesFullyDisposed = true
```

Comment:
Is this intentional? Should it be allPropertiesFullyDisposed = allPropertiesFullyDisposed?

Reply:
That is correct, it should be allPropertiesFullyDisposed = allPropertiesFullyDisposed, I have overlooked that.  As allPropertiesFullyDisposed was not within the onSubmit scope

THis should fix the validation issue